test/autoschedulers/mullapudi2016/reorder.cpp

```diff
@@ -121,7 +130,15 @@ double run_test_3(bool auto_schedule) {
         // Provide estimates on the pipeline output
         r.set_estimates({{0, 1024}, {0, 1024}, {0, 3}});
         // Auto-schedule the pipeline
-        p.apply_autoscheduler(target, {"Mullapudi2016"});
+        //
+        // Disabling this experimental GPU feature because the autoscheduler correctly
+        // identifies dimension 'c' as the stride axis for the GPU grid-stride loop,
+        // which helps retain threads efficiently. However, it fails to reorder 'c'
+        // inside the loop nests of gpu_blocks 'xo' and 'yo', which is necessary for
+        // successful code generation.
+        //
+        // Reference: https://developer.nvidia.com/blog/cuda-pro-tip-write-flexible-kernels-grid-stride-loops/
+        p.apply_autoscheduler(target, get_autoscheduler_params(false));
```

Comment:
Same two comments as above. Add a manual GPU schedule; and disable the test in the main function for now. (Don't print SKIP however, as there is more than 1 test in this file).